ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/CommandDeduplicationIT.scala

```diff
@@ -19,15 +19,17 @@ import scala.concurrent.Future
 import scala.concurrent.duration.DurationInt
 import scala.util.{Failure, Success}
 
-final class CommandDeduplicationIT extends LedgerTestSuite {
+class CommandDeduplicationIT(ledgerTimeIntervalMs: Int) extends LedgerTestSuite {
```

Comment:
Why did you remove the `final` qualifier? In which way would you like to extend this class?

Reply:
initially I did this because I was going to do a companion object but not needed now : will revert

Reply:
I'm not sure I understand. A `final` class can have a companion object.